tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
@bgruening @blankenberg Sorry, I now see that neither of these tools have been moved to the tools-devteam repo (no idea how I thought I saw that), so I'll just add them both to the tools-iuc repo.  I assume they should be in different directories...